charts/fission-core/values.yaml

```diff
@@ -269,3 +269,6 @@ mqt_keda:
     gcp_pub_sub:
       image: fission/keda-gcp-pubsub-http-connector
       tag: v0.2
+    redis:
+      image: blackfly19/keda-redislist-http-connector
```

Comment:
Please change this to `fission/keda-redis-http-connector`. Once keda connectors PR is merged, CI would build and push required image.